src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -9,6 +9,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
     Partial Friend Class BoundNode
         Implements IBoundNodeWithIOperationChildren
 
+<<<<<<< HEAD
```

Comment:
Conflict in this file is not resolved yet. I left it to @333fred since he knows better than I do :)